e2e/process-services-cloud/task-form-cloud-component.e2e.ts

```diff
@@ -79,21 +85,57 @@ describe('Task form cloud component', () => {
         done();
     }, 5 * 60 * 1000);
 
-    it('[C307032] Should display the appropriate title for the unclaim option of a Task', async () => {
+    beforeEach(() => {
         navigationBarPage.navigateToProcessServicesCloudPage();
         appListCloudComponent.checkApsContainer();
+    });
+
+    it('[C307032] Should display the appropriate title for the unclaim option of a Task', async () => {
         appListCloudComponent.goToApp(candidateBaseApp);
         tasksCloudDemoPage.myTasksFilter().clickTaskFilter();
         tasksCloudDemoPage.taskListCloudComponent().checkContentIsDisplayedByName(assigneeTask.entry.name);
-        tasksCloudDemoPage.taskListCloudComponent().selectRow(assigneeTask.entry.name);
+        tasksCloudDemoPage.taskListCloudComponent().selectRowByName(assigneeTask.entry.name);
         expect(taskFormCloudComponent.getReleaseButtonText()).toBe('RELEASE');
     });
 
+    it('[C310366] Should refresh buttons and form after an action is complete', () => {
+        appListCloudComponent.goToApp(simpleApp);
+        tasksCloudDemoPage.myTasksFilter().clickTaskFilter();
+        expect(tasksCloudDemoPage.getActiveFilterName()).toBe('My Tasks');
+        tasksCloudDemoPage.editTaskFilterCloudComponent().clickCustomiseFilterHeader().clearAssignee().setStatusFilterDropDown('CREATED');
+
+        tasksCloudDemoPage.taskListCloudComponent().checkContentIsDisplayedById(formTaskId);
+        tasksCloudDemoPage.taskListCloudComponent().selectRowById(formTaskId);
+
+        taskFormCloudComponent.checkFormIsReadOnly();
+        taskFormCloudComponent.checkClaimButtonIsDisplayed();
+        taskFormCloudComponent.checkCompleteButtonIsNotDisplayed();
+        taskFormCloudComponent.checkReleaseButtonIsNotDisplayed();
+
+        taskFormCloudComponent.clickClaimButton();
+        taskFormCloudComponent.checkSpinnerIsDisplayed();
```

Comment:
Can you remove the check for spinner please? It might fail on Travis, from time to time. At least this is what happened in some other tests that were using this check.